design-tokens-manager/src/routes/Scaler/Results.jsx

```diff
@@ -55,14 +77,66 @@ const CopyButton = ({ scale, ...rest }) => {
   );
 };
 
+const tshirtSizes = [
+  'xxxsmall',
```

Comment:
```suggestion
  '3xsmall',
```
More for future naming discussion, but right now once we hit 3 x's we shift to numeric